sdk/storage/storage-file-share/test/perfstress/track-1/upload.spec.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { Aborter, FileURL } from "@azure/storage-file";
+import { StorageFileShareTest } from "./storageTest";
+
+interface StorageFileShareUploadTestOptions {
+  size: number;
+}
+export class StorageFileShareUploadTest extends StorageFileShareTest<
+  StorageFileShareUploadTestOptions
+> {
+  public options: PerfStressOptionDictionary<StorageFileShareUploadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+
+  async runAsync(): Promise<void> {
+    const fileClient = FileURL.fromDirectoryURL(
+      this.directoryClient,
+      `newfile${new Date().getTime()}`
+    );
+    await fileClient.create(Aborter.none, this.parsedOptions.size.value!);
```

Comment:
Do we need to call both create() and uploadRange() every time, or could we call create() once then just uploadRange() in the perf test?  Unless we are explicitly trying to test both APIs together.

Reply:
Moved create to "setup"